app/controllers/forgot_password_controller.rb

```diff
@@ -0,0 +1,14 @@
+class ForgotPasswordController < ApplicationController
+  def show
+    if session[:email].blank?
+      redirect_to new_user_password_path
+    else
+      @resend_confirmation = params[:resend].present?
+      email = session.delete(:email)
+
+      @password_reset_email_form = PasswordResetEmailForm.new(email)
+
+      render :show, locals: { email: email }
```

Comment:
Good question. I was following what @jessieay did in `SignUp::EmailsController`. I'm happy to change this. I think if the view already has other instance variables, anything coming from the controller should be accessed via an instance variable in the view so that it's clear that it's an instance variable as opposed to a helper for example.